src/client/fuse_ll.cc

```diff
@@ -242,6 +248,7 @@ static int already_fuse_mounted(const char *path, bool &already_mounted)
 
   return err;
 }
+#endif
```

Comment:
Better would be to have an `#else` block that mimics old behavior for non-linux platforms. Caller need to wrapped under `#if ...` block.

Reply:
@vshankar Updated code.